.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+branches:
+  only:
+    - "master"
+
+env:
+  global:
+    - GO11MODULE=on
```

Comment:
Since we're using go 1.12 this _shouldn't_ be necessary.  Were you running into problems with the build?